src/components/graphs/piegraph/PieGraph.less

```diff
@@ -0,0 +1,5 @@
+.pie-container {
+  position: relative;
+  width: 100%;
+  min-height: 360px;
+}
```

Comment:
I'm curious, considering the above code:

`const graphOptions = {
  responsive: true,
  maintainAspectRatio: false,
};`

Did you find it helped the responsiveness of this graph by implementing the `graphOptions` _and_ making the width responsive with `width: 100%`? Not a merge-blocking thing, just curious about your thoughts here!